Synview-frontend/src/slices/projectSlice.ts

```diff
@@ -0,0 +1,18 @@
+import {createSlice} from '@reduxjs/toolkit'
+
+interface ProjectSlice{
+ 
+}
+const initialState : ProjectSlice = {
+    projects : []
+}
+
+export const projectSlice = createSlice({
+  name: "project",
+  initialState,
+  reducers : {
+    
+  }
+})
+
+export const {} = projectSlice.actions
```

Comment:
Is this just a placeholder until you add reducers? Could probably delete this line until then.